### ISSUE TITLE
[BUG] SSD degration for just having plots in it.

### ISSUE THREAD

**Issue by GeorgePapageorgakis** [Chia-Network/chia-blockchain#8541]
I have a 2TB ssd and I no longer plot with it so I have decied to just store some plots (1.8TB and 280GB free).
The problem is that its life is dropping for no apparent reason. 
Does this make any sense? Is it so write intensive to use plots for harvester?
By having plots and with less than 1% utilization I lost more than 15% in ssd life. from 90% down to 75% by just holding some plots in it in a harvester...

Desktop:
Windows 10
AMD 5950x
64gb ram
chia is always updated (1.2.7 currently)

**Comment by GeorgePapageorgakis** [comment]
Well it seems that its the windows default. I dont think that this should be a problem. Yes the firmware is updated in the latest version.

**Comment by loppefaaret** [comment]
does windows list the drive as being an SSD ? 
![image](https://user-images.githubusercontent.com/32275244/134221015-cdcd6603-7018-49d8-ac74-2e1bc0ca2180.png)
if not, it might have done defragmentation thinking it was an HDD ? 
i cant for the life of me figure out why on earth it would drop 15% life like that ? 
even a defrag shouldnt do this, unless it is going absolutely bonkers while perfoming the defrag, and the drive has a low rated TBW

**Comment by GeorgePapageorgakis** [comment]
Yes it is idetified correctly as an ssd. TRIM is enabled. Its on a x570 motherboard with amd 5950x cpu if that makes any sense. Maybe the sata controller is bad or something?

**Comment by loppefaaret** [comment]
try and take note of the amount of data written to the SSD, and watch if it grows over time - harvesting it should grow _very_ little from alone...

**Comment by loppefaaret** [comment]
> Yes it is idetified correctly as an ssd. TRIM is enabled. Its on a x570 motherboard with amd 5950x cpu if that makes any sense. Maybe the sata controller is bad or something?
> 
> https://www.tomshardware.com/news/wd-black-sn850-performance-issues-chipset-m-2

this isnt SATA based, it is NVMe based, over PCIe - is it even connected to the secondary m.2 slot on the motherboard, and does that slot lead through the X570 chipset ? 

even so, perfomance degrading would not emply additional writes to the device... it would just be slower

**Comment by GeorgePapageorgakis** [comment]
Yes its nvme wd sn850. I have 2 in two nvme slots of the motherboard. They both seem to have life drop without even plotting. its 15% over 2 months period. This is not normal.

I ll try to note down the bytes written.

**Comment by d3vnul** [comment]
> Yes its nvme wd sn850. I have 2 in two nvme slots on the motherboard. They both seem to have life drop without even plotting. its 15% over 2 months period. This is not normal.
> 
> I ll try to note down the bytes written.

I would check into windows advanced system settings, virtual memory settings. Make sure that the drive in question isn't being used for virtual memory. You may need to disable the automatic paging file size and set the nvme drive as "no paging file".

**Comment by Hardcore-fs** [comment]
Interesting fact........
(see if you can get your head around this)
current  SSD FLASH DO NOT work by "reading data"......, they work by writing charge ., even when they are "reading"

but when they are in real "write" mode they hammer the ssd harder, than "read" mode,
 becasue the real "write" includes an erase, that has to take the cell back to an "Empty" state.

Just the user does not see it...
more likely it is the quality of the SSD used.....

Simple fact...... excessive reading of SSD will kill cheap SSD.

**Comment by jmhands** [comment]
no guys...it takes about 1000 reads of a cell before firmware has to do read disturb and refresh the data. Harvester protocol is read only. You can confirm this by mounting the disk in read only mode.
I'm curious to know what is going on here. Did you setup as a swap or something? Please reach out on keybase @storage_jm I can help you poke around
